tests/02_eradiate/02_system/test_compare_rami4atm.py

```diff
@@ -145,7 +146,7 @@ def test_compare_rami4atm_onedim(
 @pytest.mark.parametrize("sza", [0.0, 30.0, 60.0])
 @pytest.mark.parametrize("lai", [1.0, 2.0, 3.0])
 def test_compare_rami4atm_rami(
-    mode_mono_double, sza, lai, artefact_dir, ert_seed_state
+    mode_ckd_double, sza, lai, artefact_dir, ert_seed_state
```

Comment:
Running this in CKD mode makes little sense IMO: the RAMI experiment is intended to be monochromatic.
```suggestion
    mode_mono_double, sza, lai, artefact_dir, ert_seed_state
```